libs/storybook/project.json

```diff
@@ -25,7 +25,10 @@
         "configDir": "libs/storybook/.storybook",
         "browserTarget": "storybook:build-storybook",
         "compodoc": false,
-        "styles": ["libs/styles/src/lib/scss/main.scss"]
+        "styles": ["libs/styles/src/lib/scss/main.scss"],
```

Comment:
Including styles to storybook nx project is necessary for the Storybook to compile if a cvi-styles import is referenced in a cvi-ng component style. Otherwise it's not needed.